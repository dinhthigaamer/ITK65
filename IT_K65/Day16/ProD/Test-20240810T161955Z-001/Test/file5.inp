432
-92666141 -538469829 -472991684 -843699349 23704508 150490716 451258008 624171807 -829185026 -491315185 -76203556 -68926616 696880327 -192280079 404308132 772574465 -722939615 858030632 -239946253 333073561 301010289 367300984 -656212366 480168068 949600446 943054987 -943297310 105017235 -70392634 -389887454 200591969 -863506681 840536237 993512613 963681985 -96208532 -282419706 -579622190 115341625 162236012 -312878779 -792284106 985596747 460288500 -825875044 604521872 -885851531 641492705 966540578 201963120 803809041 -381646699 -477104372 508013173 -939378260 -2893215 -954807376 -20129935 -48361341 -6344404 -910805388 430910689 796838524 24348124 -755145846 -659350911 -746539379 -683353808 -213997410 -61852025 -571777587 -598010065 -465145922 602706043 -158023424 -19423407 -534147774 -421882880 569507294 441246334 688112139 -238981406 20932413 295573252 -830616793 -866099730 417150783 345857912 943663440 620866679 422714686 -188416496 190264522 -608951448 948721140 -727491851 -683416594 -568662164 853257102 825204146 -979103434 303434456 541136952 -273528346 -382212546 927876471 -720073444 287218234 324088712 -559844811 -358640098 221262965 840969832 -727301018 -40649799 145050475 922146727 895086620 -360444279 -966866783 410928560 618091499 97784037 916840506 -234439480 -177563037 435433036 -77380535 102617644 -80955149 779894931 -411368533 266342657 611222134 -30889792 1293662 -100336333 -38672046 848837414 -458392745 -313921309 -183208891 -442379533 208076853 -349267263 -557176668 773150891 -722827814 -804665202 -221998087 353742789 -774254939 -97523221 952167862 600080702 456992982 581663741 -636266064 -398974980 -918562773 -245600070 -774314204 -437750929 51735565 386274463 -760264949 407368980 347243601 -794612864 -821889712 186099009 -449929721 211607144 990324703 174082464 225370972 245295756 -129070593 -772047237 391057427 320486941 -688811519 346390680 -899533165 -859328609 -958003811 -722705802 -86354909 -491367765 381144958 534703151 -498746257 177629793 476735665 475174788 78534856 -821420478 542641971 518394880 -514894489 -89761597 828442269 -918104497 -727403253 -362816499 -530956276 -237702061 644154792 -625938811 -340713549 -342067718 -325254900 -865159262 -690396961 -474413658 -35038106 -236351499 281731415 -284670496 -336735006 -260396599 149251905 714334020 940057054 -752617529 604578772 -905371741 969314057 895449935 -619191091 -635788887 -195070062 -440503430 -912471383 -618306211 92017420 686148189 -245198105 398868582 995720724 341202934 -327903612 -78983680 574120259 -297397163 -778539155 520002787 531378192 -904467228 919784680 31377696 121271419 929461728 -627684192 -876326563 578539175 -902074025 184666381 9684413 -467324610 -408063147 -952046546 138305438 -624798140 125053320 -671755215 -605369074 771632123 165170954 65242028 586501012 231384825 -479359421 -61301338 86882001 -418440175 23460467 -41707350 -874437180 356632810 937923052 -57202802 -459944138 -682544476 -5265210 442922023 -233822041 -634340951 -135294054 -141777695 -510496866 177005961 -481842205 891556670 -513928654 -663578994 -981348595 603203031 -265515969 79323627 -804736548 -255111770 -50543390 910782199 -645773706 -307783771 430981232 -510663622 546190790 272682211 -573472134 -243883391 -854356699 -509003036 -580738708 -433066925 46453063 -691574290 423894023 -2207097 -320956218 473008852 273163351 -28662506 701299897 -727604888 173662052 -721901275 851127613 -336844043 253069616 238021678 143004029 588495093 -605634013 64587483 -815779190 163695385 -123801764 384071803 -432390507 549490933 926210259 -917890622 -826563648 -52019466 235274521 4880276 674523415 -591309041 210405062 -962343240 757117936 241958488 -405334485 -543557951 397795313 255330041 -249829778 133113521 -400418322 654359915 176332405 405375835 825611565 -723471800 -194169552 -873324739 -886152398 986726979 567167634 -98156841 -801517982 742636618 232057892 747476355 40796099 998033470 391462475 -538256567 125380799 -865827062 365148780 903097833 -877770670 -668712935 462501077 -965463244 869789284 92300879 898980080 763920014 146064046 22032057 87920092 124371138 631680460 636994788 -408940164 -998901964 -955973645 -249801469 -637892602 -496992012 -625058673 -893649987 92252553 980621934 -944722094 -901814943 9867344 797121469 -244960551 852816391 668731568 -218240447 -135989198 798570279 66927902 295902028 736813059 -203685776 407959627 -316727829 564949146 736129793 52226968 -273264856 126678859 797148432 824675006 379923744 
-92666141 -538469829 -472991684 -843699349 23704508 150490716 451258008 624171807 -829185026 -491315185 -76203556 -68926616 696880327 -192280079 404308132 772574465 -722939615 858030632 -239946253 333073561 301010289 367300984 -656212366 480168068 949600446 943054987 -943297310 105017235 -70392634 -389887454 200591969 -863506681 840536237 993512613 963681985 -96208532 -282419706 -579622190 115341625 162236012 -312878779 -792284106 985596747 460288500 -825875044 604521872 -885851531 641492705 966540578 201963120 803809041 710927939 -477104372 508013173 -939378260 -2893215 -954807376 -20129935 -48361341 -6344404 -910805388 430910689 796838524 24348124 -755145846 -659350911 -746539379 -683353808 -213997410 -61852025 -571777587 -598010065 -465145922 602706043 -158023424 -19423407 -534147774 -421882880 569507294 441246334 688112139 -238981406 20932413 295573252 -830616793 -866099730 417150783 345857912 943663440 620866679 422714686 -188416496 190264522 -608951448 948721140 -727491851 -683416594 -568662164 853257102 825204146 -979103434 303434456 541136952 -273528346 -382212546 927876471 -720073444 287218234 324088712 -559844811 -358640098 221262965 840969832 -727301018 -40649799 145050475 922146727 895086620 -360444279 -966866783 410928560 618091499 97784037 916840506 -234439480 -177563037 435433036 -77380535 102617644 -80955149 779894931 -411368533 266342657 611222134 -30889792 1293662 -100336333 -38672046 848837414 -458392745 -313921309 -183208891 -442379533 208076853 -349267263 -557176668 773150891 -722827814 -804665202 -221998087 353742789 -774254939 -97523221 952167862 600080702 456992982 581663741 -636266064 -398974980 -918562773 -245600070 -774314204 -437750929 51735565 386274463 -760264949 407368980 347243601 -794612864 -821889712 186099009 -449929721 211607144 990324703 174082464 225370972 245295756 -129070593 -772047237 391057427 320486941 -688811519 346390680 -899533165 -859328609 -958003811 -722705802 -86354909 -491367765 381144958 534703151 -498746257 177629793 476735665 475174788 78534856 -821420478 542641971 518394880 -514894489 -89761597 828442269 -918104497 -727403253 -362816499 -530956276 -237702061 644154792 -625938811 -340713549 -342067718 -325254900 -865159262 -690396961 -474413658 -35038106 -236351499 281731415 -284670496 -336735006 -260396599 149251905 714334020 940057054 -752617529 604578772 -905371741 969314057 895449935 -619191091 -635788887 -195070062 -440503430 -912471383 -618306211 92017420 686148189 -245198105 398868582 995720724 341202934 -327903612 -78983680 574120259 -297397163 -778539155 520002787 531378192 -904467228 919784680 31377696 121271419 929461728 -627684192 -876326563 578539175 -902074025 184666381 9684413 -467324610 -408063147 -952046546 138305438 -624798140 125053320 -671755215 -605369074 771632123 165170954 65242028 586501012 231384825 -479359421 -61301338 86882001 -418440175 23460467 -41707350 -874437180 356632810 937923052 -57202802 -459944138 -682544476 -5265210 442922023 -233822041 -634340951 -135294054 -141777695 -510496866 177005961 -481842205 891556670 -513928654 -663578994 -981348595 603203031 -265515969 79323627 -804736548 -255111770 -50543390 910782199 -645773706 -307783771 430981232 -510663622 546190790 272682211 -573472134 -243883391 -854356699 -509003036 -580738708 -433066925 46453063 -691574290 423894023 -2207097 -320956218 473008852 273163351 -28662506 701299897 -727604888 173662052 -721901275 851127613 -336844043 253069616 238021678 143004029 588495093 -605634013 64587483 -815779190 163695385 -123801764 384071803 -432390507 549490933 926210259 -917890622 -826563648 -52019466 235274521 4880276 674523415 -591309041 210405062 -962343240 757117936 241958488 -405334485 -543557951 397795313 255330041 -249829778 133113521 -400418322 654359915 176332405 405375835 825611565 -723471800 -194169552 -873324739 -886152398 986726979 567167634 -98156841 -801517982 742636618 232057892 747476355 40796099 998033470 391462475 -538256567 125380799 -865827062 365148780 903097833 -877770670 -668712935 462501077 -965463244 869789284 92300879 898980080 763920014 146064046 22032057 87920092 124371138 631680460 636994788 -408940164 -998901964 -955973645 -249801469 -637892602 -496992012 -625058673 -893649987 92252553 980621934 -944722094 -901814943 9867344 797121469 -244960551 852816391 668731568 -218240447 -135989198 798570279 66927902 295902028 736813059 -203685776 407959627 -316727829 564949146 736129793 52226968 -273264856 126678859 797148432 824675006 379923744 