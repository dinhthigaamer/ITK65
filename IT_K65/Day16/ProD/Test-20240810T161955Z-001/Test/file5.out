-5264477398967
